src/store/metadata/stub/content/editions/versions/additions-to-the-housing-stock_2022-2023.json

```diff
@@ -0,0 +1,125 @@
+            "issued": "2023-05-05T08:59:11Z",
+            "title": "Net additions to the housing stock",
+            "summary": "Net additions to the dwelling stock (including new builds, conversions, changes of use, other gains/losses and offset by demolitions) per 1,000 stock, England, financial year 2022 - 2023.",
+            "description": "This dataset shows the net additions to the dwelling stock (including new builds, conversions, changes of use, other gains/losses and offset by demolitions) per 1,000 stock, in England, for financial year 2022 - 2023.\n\n### Coverage \n\nEngland\n\n#### Geography definition\n\nLocal Authority Districts and Unitary Authorities,\nCounties,\nMetropolitan Counties,\nNation (England) \n\n### Source\n\n - England: [Department for Levelling Up, Housing and Communities]",
+            "download_url": "https://staging.idpd.uk/datasets/additions-to-the-housing-stock/editions/2022-2023/versions/1.csv",
+            "media_type": "text/csv",
+            "table_schema": {
+          "columns": [
+            {
+                "name": "Period",
+                "datatype": "string",
+                "titles": "Period",
+                "description": "The time period associated with the observation."
+              },
+              {
+                "name": "AREACD",
+                "datatype": "string",
+                "titles": "Statistical Geography",
+                "description": "The geography code associated with the observation."
+              },
+              {
+                "name": "AREANM",
+                "datatype": "string",
+                "titles": "Area Name",
+                "description": "The name assiciated with The geography code "
+              },
+              {
+                "name": "Geography",
+                "datatype": "string",
+                "titles": "Geography Type",
+                "description": "The categorisation of the geography code"
+              },
+              {
+                "name": "Variable Name",
+                "datatype": "string",
+                "titles": "Variable Name",
+                "description": "Which ESS variable is being measured"
+              },
+              {
+                "name": "Indicator", 		
+                "datatype": "string",
+                "titles": "Indicator",
+                "description": "Which ESS indicator is being measured"
+              },
+              {
+                "name": "Mission",
+                "datatype": "string",
+                "titles": "Mission",
+                "description": "Which ESS mission number this dataset is fulfilling"
+              },
+              {
+                "name": "Category",
+                "datatype": "string",
+                "titles": "Category",
+                "description": "Which ESS category the dataset belongs to"
+              },
+              {
+                  "name": "Observation",
+                  "datatype": "integer",
```

Comment:
datatype for Observation should be `"decimal"` just like in `apprenticeships-achievements_2023-05.json`